index.html

```diff
@@ -841,9 +841,9 @@
               </li>
               <li>Determine which <a>payment handlers</a> support
               <var>identifier</var>. For each resulting payment handler, if
-              payment method specific capabilities supplied by the payment
-              handler match those provided by <var>data</var>, the payment
-              handler matches.
+              <var>data</var> contains filters that exclude the payment
```

Comment:
I still find the word "filter" extremely problematic. The `data` attribute contains only "members" - and those members are acted upon based on the specification that defines those member (or no action is taken, in the case the PMI is a URL-based PMI).